client/src/components/lobby/Lobby.js

```diff
@@ -28,26 +37,38 @@ const Lobby = (): React$Element<any> => {
     });
   }, [socket]);
 
-  const firstUpdate = useRef(true);
   useEffect(() => {
```

Comment:
TODO: Refactor to state machine
TODO: Refactor UsersList management to a different component, sort of a Repository Pattern